[2025-08-24T09:40:02.892-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-24T09:40:02.916-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_processamento_viagens.executar_etl_de_viagens scheduled__2025-08-24T12:30:00+00:00 [queued]>
[2025-08-24T09:40:02.920-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_processamento_viagens.executar_etl_de_viagens scheduled__2025-08-24T12:30:00+00:00 [queued]>
[2025-08-24T09:40:02.920-0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-24T09:40:02.928-0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): executar_etl_de_viagens> on 2025-08-24 12:30:00+00:00
[2025-08-24T09:40:02.936-0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_processamento_viagens', 'executar_etl_de_viagens', 'scheduled__2025-08-24T12:30:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/dag_processamento_viagens.py', '--cfg-path', '/tmp/tmpjejnnfw9']
[2025-08-24T09:40:02.938-0300] {standard_task_runner.py:91} INFO - Job 91: Subtask executar_etl_de_viagens
[2025-08-24T09:40:02.939-0300] {logging_mixin.py:188} WARNING - /home/caioa/Airflow/venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=25682) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-24T09:40:02.939-0300] {standard_task_runner.py:63} INFO - Started process 25696 to run task
[2025-08-24T09:40:02.966-0300] {task_command.py:426} INFO - Running <TaskInstance: pipeline_processamento_viagens.executar_etl_de_viagens scheduled__2025-08-24T12:30:00+00:00 [running]> on host PCzo.localdomain
[2025-08-24T09:40:03.009-0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pipeline_processamento_viagens' AIRFLOW_CTX_TASK_ID='executar_etl_de_viagens' AIRFLOW_CTX_EXECUTION_DATE='2025-08-24T12:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-24T12:30:00+00:00'
[2025-08-24T09:40:03.010-0300] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-24T09:40:03.023-0300] {logging_mixin.py:188} INFO - --- Iniciando Pipeline de ETL de Viagens ---
[2025-08-24T09:40:03.023-0300] {logging_mixin.py:188} INFO - Arquivo de origem não encontrado ou vazio: /home/caioa/Airflow/data/novas_avaliacoes.csv
[2025-08-24T09:40:03.023-0300] {logging_mixin.py:188} INFO - Nenhum dado novo para processar.
[2025-08-24T09:40:03.024-0300] {logging_mixin.py:188} INFO - --- Pipeline de ETL de Viagens Concluído ---
[2025-08-24T09:40:03.024-0300] {python.py:237} INFO - Done. Returned value was: None
[2025-08-24T09:40:03.024-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-24T09:40:03.026-0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_processamento_viagens, task_id=executar_etl_de_viagens, run_id=scheduled__2025-08-24T12:30:00+00:00, execution_date=20250824T123000, start_date=20250824T124002, end_date=20250824T124003
[2025-08-24T09:40:03.073-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-24T09:40:03.080-0300] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-24T09:40:03.081-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
